packages/components/message-box/src/index.vue

```diff
@@ -173,6 +173,7 @@ import {
 import ElInput from '@element-plus/components/input'
 import { ElOverlay } from '@element-plus/components/overlay'
 import {
+  CloseComponents,
```

Comment:
There is no need to import it separately here, because `TypeComponents` already contains the `close` component.

Reply:
You are right. The `components` already contain the `close` component, and I can use it via a string.